### PR TITLE
Add NeutronRC F7 AIO

### PR DESCRIPTION
**Pull request by DusKing1**
AIO board, ESCs are onboard. So we will take response for the Dshot settings

image

### CODE REVIEW

**Comment by DusKing1** [comment]
@github-actions get away

File: configs/default/NEUT-NEUTRONRCF7AIO.config [hunk 1]
```diff
@@ -0,0 +1,147 @@
+# Betaflight / STM32F745 (S745) 4.1.0 Sep 25 2019 / 01:23:16 (064ca9b75) MSP API: 1.42
+
+board_name NEUTRONRCF7AIO
+manufacturer_id NEUT
```
Comment (mikeller) [inline configs/default/NEUT-NEUTRONRCF7AIO.config:4]: There does not seem to be a manufacturer with this ID. Can you please follow [these instructions](https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md#23-add-the-board-and-manufacturer-information) to get one added?

File: configs/default/NEUT-NEUTRONRCF7AIO.config [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+feature OSD
+feature ESC_SENSOR
+
+# serial
+serial 0 8192 115200 57600 0 115200
+serial 1 64 115200 57600 0 115200
+serial 5 1024 115200 57600 0 115200
+
+# aux
+aux 0 0 0 900 1300 0 0
```
Comment (mikeller) [inline configs/default/NEUT-NEUTRONRCF7AIO.config:120]: This should not be in here.

File: configs/default/NEUT-NEUTRONRCF7AIO.config (hunk 1 above)
Comment (DusKing1) [reply configs/default/NEUT-NEUTRONRCF7AIO.config:4]: OMG, I haven't awared that my ID is no longer the NEUT but NERC

File: configs/default/NEUT-NEUTRONRCF7AIO.config (hunk 2 above)
Comment (DusKing1) [reply configs/default/NEUT-NEUTRONRCF7AIO.config:120]: Will fix this

**Review by mikeller — APPROVED** [review]

**Pull request merged**